apps/hash-api/src/graphql/resolvers/knowledge/org/accept-org-invitation.ts

```diff
@@ -186,9 +184,8 @@ export const acceptOrgInvitationResolver: ResolverFn<
   if (!creatorIsOrgAdmin) {
     await archiveInvitation();
 
-    throw new ApolloError(
+    throw Error.forbidden(
       "Invitation issuer is not an administrator of the organization",
-      "UNAUTHORIZED",
     );
```

Comment:
This is technically a behaviour change, but the codebase is already incorrectly using `FORBIDDEN` for both unauthenticated (i.e. `UNAUTHORIZED` in http terms, boo) and authenticated but not authorized (`FORBIDDEN`, which it should be in this case, so this change is fine). I don't suggest we try and fix that conflation in this PR.